tests/test-conf.yaml

```diff
@@ -8,3 +8,11 @@ dispatcher:
     logstash_host: 10.194.169.75
     logstash_port: 5001
     secret_key: 'secretkey_test'
+    smtp_server: 'localhost'
+    sender_mail: 'team@odahub.io'
+    cc_receivers_mail: ['team@odahub.io']
+    smtp_port: 1025
+    smtp_server_password: ''
+    mail_sending_timeout: True
+    mail_sending_timeout_threshold: 1800
```

Comment:
I would call it `mail_sending_timeout_default_threshold` for clarity. It's how it is used, yes?

Also, small suggestion, why not make another dict layer for mail, for example? There are quite many keys. It will be more readable.

Reply:
by the way, there is example config there in conf_dir, it would need to be updated, although it does not affect immediately any functionality:

https://github.com/oda-hub/dispatcher-app/blob/master/cdci_data_analysis/config_dir/conf_env.yml.example

Reply:
good idea, configuration options are growing